liota/dccs/iotcc.py

```diff
@@ -287,6 +305,57 @@ def prettify(self, elem):
         reparsed = minidom.parseString(rough_string)
         return reparsed.toprettyxml(indent="    ")
 
+    def _init_info(self):
+        msg = {
+            "iotcc": {
+                "EdgeSystem": {"SystemName": "", "EntityType": "", "uuid": ""},
+                "OGProperties": {"OrganizationGroup": ""},
```

Comment:
Do we have multiple properties or only Organization Group Name? It can be confusing why we need OGProperties if it is just the Org Group name.